### DISCUSSION TITLE
Problem with plugin installation

### DISCUSSION THREAD

**Discussion by labropu** [netbox-community/netbox#12574] [Help Wanted!] [answered]
Hello,

I have been using netbox for a few weeks now and had already installed some plugins.

Today I tried installing netbox-ddns and netbox-dns plugins but after installing, when i executed `./manage.py migrate` I received the following error:
`django.core.exceptions.ImproperlyConfigured: Unable to import plugin nextbox_ui_pluginnetbox_ddnsnetbox_dns: Module not found. Check that the plugin module has been installed within the correct Python environment.`

Executing the pip list command under virtual environment I can see the mentioned packages installed.

Moreover, after doing a service restart, netbox doesn't operate at all and I have the following status:
```
nbuser@netbox:~$ sudo systemctl status netbox
● netbox.service - NetBox WSGI Service
     Loaded: loaded (/etc/systemd/system/netbox.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Fri 2023-05-12 11:49:02 UTC; 9s ago
       Docs: https://netbox.readthedocs.io/en/stable/
    Process: 2552 ExecStart=/opt/netbox/venv/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/netbox --config /opt/netbox/gunicorn.py netbox.wsgi (code=exited, status=1/FAILURE)
   Main PID: 2552 (code=exited, status=1/FAILURE)
```

Please your help.

**Comment by a084ed22** [comment, marked as answer]
Check on your configuration.py for the plugin name definition, it seems like all their names got squashed into a single string rather than being distinct names

**Reply by labropu** [reply]
You are absolutely correct. I think I doublechecked this file but I never saw that a few commas were missing... :)

Thank you very much.
